concept.tex

```diff
@@ -449,9 +449,17 @@ \subsubsection{Open Source ecosystem}\label{sec:opensource}\TODO{Should the
 repository.
 
 \emph{\mybinder{}} (see \ref{sec:mybinder}) is a service provided by the \myemph{BinderHub
-  federation} that collectively host a service running the Binder software
-under the URL \url{https://mybinder.org}. This is the service we made use of in
-our example in section~\ref{sec:reproducibility-example}.
+Federation} that collectively host a service running the Binder software
+under the URL \url{https://mybinder.org}. (This is the service we made use of in the example
+of section~\ref{sec:reproducibility-example}.)
+
+The BinderHub Federation is currently composed of three deployments of the BinderHub software
+
+\begin{itemize}
+\item \url{https://gke.mybinder.org}, operated by the Binder team, and hosted on Google Cloud,
+\item \url{https://ovh.mybinder.org}, operated and hosted by OVH,
+\item \url{https://gesis.mybinder.org}, operated and hosted by GESIS.
```

Comment:
I'd still mention Turing, which paused a few weeks ago while funding is renewed, but still an active member. Might be worth mentioning KAUST, which is currently being onboarded: https://github.com/jupyterhub/mybinder.org-deploy/issues/1772 (only blocker is me focusing on this grant!)